v3/src/components/web-view/web-view-registration.ts

```diff
@@ -0,0 +1,26 @@
+import { registerTileComponentInfo } from "../../models/tiles/tile-component-info"
+import { registerTileContentInfo } from "../../models/tiles/tile-content-info"
+import { kWebViewTileType } from "./web-view-defs"
+import { WebViewModel } from "./web-view-model"
+import { WebViewComponent } from "./web-view"
+import { WebViewInspector } from "./web-view-inspector"
+import { WebViewTitleBar } from "./web-view-title-bar"
+
+export const kWebViewIdPrefix = "WEBVIEW"
```

Comment:
By convention we generally keep these to four characters, so I'd probably go with:
```suggestion
export const kWebViewIdPrefix = "WEBV"
```